### PR TITLE
Rename scripts

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-845/1/

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-845/3/

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/tests/test_plot_stats_per_point.py [hunk 1]
```diff
@@ -13,15 +13,15 @@
 
 
 def test_help_option(script_runner):
-    ret = script_runner.run('scil_plot_mean_std_per_point.py', '--help')
+    ret = script_runner.run('scil_plot_stats_per_point.py', '--help')
```
Comment (EmmaRenauld) [inline scripts/tests/test_plot_stats_per_point.py:16]: scil_visualize???

**Review by EmmaRenauld — APPROVED** [review]

**Pull request merged**